en/docs/develop/product-apis/admin-apis/admin-v1/admin-v1.yaml

```diff
@@ -2606,9 +2891,13 @@ paths:
       x-wso2-curl: "curl -k -X POST -H \"Authorization: Bearer 0d63e133-7ad6-3aeb-9ca9-9299e0708122\" -H \"Content-Type: application/json\" https://apis.wso2.com/api/am/admin/v1/labels -d @data.json"
       x-wso2-request: "POST https://localhost:9443/api/am/admin/v1/labels\r\nAuthorization: Bearer 0d63e133-7ad6-3aeb-9ca9-9299e0708122\r\nContent-Type: application/json\r\n\r\n   {\r\n       \"name\":\"internal\",\r\n	\"description\":\"label used for internal purpose\",\r\n	\"accessUrls\":[\r\n	  \"https://localhost:9095\"\r\n         ]\r\n     }"
```

Comment:
Shall we remove x-wso2-curl,  x-wso2-request, x-wso2-response extensions, since they are not in use?

Reply:
fixed with https://github.com/wso2/docs-apim/pull/1842/commits/09f97e448ff264ecc0b6025cb1c9ac19f2228951